scipy/stats/_continuous_distns.py

```diff
@@ -5289,7 +5289,7 @@ class kappa4_gen(rv_continuous):
 
     """
     def _argcheck(self, h, k):
-        return h == h
+        return np.full((h*k).shape, fill_value=True)
```

Comment:
This works, and I wouldn't mind a hack like this if, say, one of the two arrays were masked. But to communicate the intent and avoid unnecessary multiplication:
```suggestion
        shape = np.broadcast_arrays(h, k)[0].shape
        return np.full(shape, fill_value=True)
```
After we drop support for Numpy 1.19 on [June 21, 2022](https://numpy.org/neps/nep-0029-deprecation_policy.html), even better would be:
```suggestion
        shape = np.broadcast_shapes(h.shape, k.shape)
        return np.full(shape, fill_value=True)
```

Reply:
If I get to it before you, I'll push this and merge when tests pass.